### DISCUSSION TITLE
Be able to enable swc from next.config

### DISCUSSION THREAD

**Discussion by jlopezxs** [vercel/next.js#32951] [Ideas]
### Describe the feature you'd like to request

Right now I'm using babel plugins that have no equivalent in swc (transform-react-qa-classes) but I don't need it for development enviroment.
In the future the idea is create a custom swc plugin for that, but now nextjs don't support custom plugins. But for now we would like to be able to enable SWC only in dev.

### Describe the solution you'd like

Currently the only validation of SWC is that there is a babel configuration.
` / packages / next / build / webpack-config.ts # L417`

Could you add one more setting in next.config to force swc?

### Describe alternatives you've considered

Wait for be able to attach custom swc plugins.

**Comment by balazsorban44** [comment]
You could easily achieve this by a script that toggles the name of a `.babelrc` file since that is what Next.js detects.

**Reply by vaniyokk** [reply]
> You could easily achieve this by a script that toggles the name of a `.babelrc` file since that is what Next.js detects.

This will affect git working copy. It'd be nice to have non-intrusive option.
